### PR TITLE
FLUT-927176 - [Others] Added tutorial samples in user guide

### PR DESCRIPTION
**Pull request by sfHariHaraSudhan**
## Description ##

Updated the published video tutorial samples in user guide on the below pages.

**SfCartesianChart** - How to Create Flutter Charts from JSON Data [[Getting started with Flutter Cartesian Charts widget | Syncfusion](https://help.syncfusion.com/flutter/cartesian-charts/getting-started#bind-data-source)]
image

**SfCalendar** - How to Display More Appointments in the Flutter Calendar [For this, I thought, we can add in [appointments page.](https://help.syncfusion.com/flutter/calendar/appointments)]
image

**SfCalendar**  - How to Customize the Date Picker's Appearance in the Flutter Calendar [[Date Navigations in Flutter Event Calendar widget | Syncfusion](https://help.syncfusion.com/flutter/calendar/date-navigations#show-date-picker)] 
image

### CODE REVIEW

**Review by LavanyaGowtham2021 — CHANGES_REQUESTED** [review]

File: Flutter/calendar/date-navigations.md [hunk 1]
```diff
@@ -213,6 +213,11 @@ Widget build(BuildContext context) {
 
 ![Show date picker](images/date-navigation/show_date_picker.png)
 
+To know more about how to customize the Date Picker's ppearance in the Flutter Calendar, you can check on this video.
```
[the remark this answers is not in the document]
Comment (sfHariHaraSudhan) [reply Flutter/calendar/date-navigations.md:216]: Hi,
Resolved this spelling mistakes.
Regards,
Hari Hara Sudhan. K.

File: Flutter/calendar/appointments.md [hunk 2]
```diff
@@ -1168,6 +1168,11 @@ Gets the pattern appointment for the specified occurrence by using the [getPatte
 
 ## See also
 
+To know more about how to display more appointments in the Flutter Calendar, you can check on this video.
+
+<style>#flutterChartVideoTutorial{width : 90% !important; height: 300px !important }</style>
```
[the remark this answers is not in the document]
Comment (sfHariHaraSudhan) [reply Flutter/calendar/appointments.md:1173]: Hi,
Missed to update it properly at first. Modified now properly.
Regards,
Hari Hara Sudhan. K.

File: Flutter/calendar/appointments.md [hunk 3]
```diff
@@ -1168,6 +1168,11 @@ Gets the pattern appointment for the specified occurrence by using the [getPatte
 
 ## See also
 
+To know more about how to display more appointments in the Flutter Calendar, you can check on this video.
```
[the remark this answers is not in the document]
Comment (sfHariHaraSudhan) [reply Flutter/calendar/appointments.md:1171]: Hi,
Modified the content as mentioned.
Regards,
Hari Hara Sudhan. K.

File: Flutter/calendar/date-navigations.md [hunk 4]
```diff
@@ -213,6 +213,11 @@ Widget build(BuildContext context) {
...
+
+<style>#flutterChartVideoTutorial{width : 90% !important; height: 300px !important }</style>
```
[the remark this answers is not in the document]
Comment (sfHariHaraSudhan) [reply Flutter/calendar/date-navigations.md:218]: Hi,
Missed to update it properly at first. Modified now properly.
Regards,
Hari Hara Sudhan. K.

**Review by LavanyaGowtham2021 — APPROVED** [review]
Fine

**Pull request merged**